tailscale/rootfs/etc/s6-overlay/scripts/stage2_hook.sh

```
@@ -5,6 +5,61 @@
 # S6 Overlay stage2 hook to customize services
 # ==============================================================================
 
+declare options
+declare proxy funnel proxy_and_funnel_port
+declare share_homeassistant share_on_port
+
+# This is to execute potentially failing supervisor api functions within conditions,
+# where set -e is not propagated inside the function and bashio relies on set -e for api error handling
+function try {
+    set +e
+    (set -e; $@)
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)